### PR TITLE
Sound title and Description redesigned

### LINKED ISSUE #51: ui: redesign implementation: sound title redesign
Redesign the current sound title to match the UI design

Screen Shot 2020-09-19 at 1 24 49 PM

### PR DESCRIPTION
**Pull request by akshara-sun**
closes #51

### CODE REVIEW

**Review by amilajack — COMMENTED** [review]
I like how you're using multiple commits to keep track of the changes. That makes it easy to see what was changed 💯

**Comment by amilajack** [comment]
You have some merge conflicts because Hamir's changes just got merged. You can click the "resolve conflicts" button to fix this.

**Comment by akshara-sun** [comment]
@amilajack for sure, I'll take care of those merge conflicts! Thanks!

File: src/styles.css [hunk 1]
```diff
@@ -31,10 +31,9 @@ h1, h1, h3, h4, h5, h6 {
   text-align: center;
 }
 
-.bg-default {
+/* .bg-default {
```
Comment (amilajack) [inline src/styles.css:34]: Why comment this?

File: src/styles.css [hunk 2]
```diff
@@ -9,10 +9,10 @@
     --secondary-color:#FBDC57;
   }
 }
-
+/*
```
Comment (amilajack) [inline src/styles.css:12]: Why comment this?

**Review by amilajack — CHANGES_REQUESTED** [review]

**Pull request merged**